packages/workspace/project.json

```diff
@@ -87,12 +87,6 @@
           },
           {
             "command": "node ./scripts/add-dependency-to-build.js workspace nx"
-          },
-          {
-            "command": "node ./scripts/add-dependency-to-build.js workspace @nrwl/devkit"
```

Comment:
`@nrwl/devkit` is already listed in the `package.json` dependencies, no need for the script